.circleci/config.yml

```diff
@@ -19,10 +19,10 @@ jobs:
                   sudo npm install -g npm@6
 
                   cp server/config/local.json-dist server/config/local.json
-                  npm i --production
+                  npm ci
 
                   CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build
-                  npm install intern xmlhttprequest firefox-profile request convict yargs leadfoot otplib retry-cli
```

Comment:
this causes issues

![](http://v14d.com/i/d5ed22fa455107dca7e29d27ceed9666dc03ed0cb8c6dbe7aa5fd579881ee125.jpg)